report/templates/activity.js

```diff
@@ -3,29 +3,26 @@ const recent_activity = {{recent_activity}}
 // Setup the chart
 nv.addGraph(function() {
 	var chart = nv.models.historicalBarChart().options(recent_activity.config);
-	chart.yAxis.options(recent_activity.yAxis);
-	chart.xAxis.options(recent_activity.xAxis);
+	chart.yAxis.options({axisLabel: "Commits"});
```

Comment:
for most options you don't really need to go through .options() then.
`chart.yAxis.axisLabel("Commits");` should work too. I had used options with data in the json file to keep as few logic as possible in the javascript files, but maybe having the settings on JS side is simpler after all.

Reply:
Thanks for the hint. I still need to learn about nvd3.
The idea of the PR was to make htmlreportcreator.py-file at least a little more structured with least efforts.
I am really grateful for your work on graphs and thanks to the work you have done, repostat is now in PyPi